util_runner.go

```diff
@@ -168,3 +179,17 @@ func createPidFile(path string, process *libcontainer.Process) error {
 	}
 	return os.Rename(tmpName, path)
 }
+
+func adjustOomScore(process *libcontainer.Process) error {
+	pid, err := process.Pid()
+	if err != nil {
+		return err
+	}
+	oomScoreAdjPath := path.Join("/proc/", strconv.Itoa(pid), "oom_score_adj")
+	value := strconv.Itoa(*process.OOMScoreAdj)
```

Comment:
Let's add a comment to the function that specifies that `process.OOMScoreAdj` cannot be nil, or add a check for `OOMScoreAdj` if nil before dereferencing the pointer. I prefer the later.